### PR TITLE
Added ADDA

### PR DESCRIPTION
**Pull request by bpas247**
Addresses issue #8.

### CODE REVIEW

File: easier68k/core/opcodes/adda.py [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+
+    # Allowed sizes for this opcode
+    valid_sizes = [OpSize.WORD, OpSize.LONG]
+
+    def __init__(self, params: list, size: OpSize = OpSize.WORD):
+        assert len(params) == 2
+        assert isinstance(params[0], AssemblyParameter)
+        assert isinstance(params[1], AssemblyParameter)
+        # Can't take data register, address register, or ARI with modifications
+        # assert params[0].mode not in [EAMode.DRD, EAMode.ARD, EAMode.ARIPD, EAMode.ARIPI, EAMode.IMM]
```
Comment (Chris-Johnston) [inline easier68k/core/opcodes/adda.py]: Unintentional commented out code?

Comment (bpas247) [reply easier68k/core/opcodes/adda.py]: The source destination is an effective address that allows for all modes so this can just be deleted.

**Comment by bpas247** [comment]
What else should be done for this PR? Is it ready to merge?

**Comment by Chris-Johnston** [comment]
LGTM :shipit:

**Pull request merged**